guides/source/maintenance_policy.md

```diff
@@ -59,6 +59,11 @@ be built from 1.2.2, and then added to the end of 1-2-stable. This means that
 security releases are easy to upgrade to if you're running the latest version
 of Rails.
 
+Only direct security patches will be included in security releases. Fixes to address
+non-security-related bugs or other implications of a security patch may be published
+on a series' X-Y-stable branch on git, but will be only be released as a new gem in
```

Comment:
```suggestion
on a release's x-y-stable branch, and will be only be released as a new gem in
```
To match the style of the paragraph above.

Reply:
I agree with most of this suggestion, but I thought the capitalised `X-Y-stable` branch name better matched the style of the rest of the document?